azure.yaml

```diff
@@ -34,7 +34,7 @@
     - name: cert-operator
       version: 0.1.0
     - name: chart-operator
-      version: 0.7.0
+      version: 0.11.4
```

Comment:
@rossf7 I haven't changed this, not sure yet who did, nevertheless I'm curious and was hoping maybe you can help me understand - why did we keep referencing chart-operator 0.7.0 so far and not actual chart-operator GitHub releases? I'm trying to figure out if this change should be done or not.

Reply:
> not sure yet who did, nevertheless I'm curious and was hoping maybe you can help me understand - why did we keep referencing chart-operator 0.7.0 so far and not actual chart-operator GitHub releases?

It was us, @giantswarm/team-celestial - The thing is, so far the `chart-operator` version for tenant cluster always came from `cluster-operator` and the line here was just for general awareness.

It was apparently wrong, as we were able to see it and hence we brought it up to date.

@sslavic Do you think this is wrong?

Reply:
@tuommaki Yes this is wrong. chart-operator should only be in apps and not in authorities. This got removed but I think only for AWS.

In tenant clusters chart-operator is deployed as an app. In control plane it is deployed by draughtsman and shouldn't be in the release.

Reply:
I just removed chart-operator from authorities and bump it to 0.11.4 in apps, @rossf7 PTAL

Reply:
@sslavic Great thanks for updating.